library/gui/key_bindings/src/main/java/org/quiltmc/qsl/key/bindings/impl/ConflictTooltipOwner.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.key.bindings.impl;
+
+import java.util.List;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+import net.minecraft.text.Text;
+
+@Environment(EnvType.CLIENT)
+public interface ConflictTooltipOwner {
```

Comment:
This could either be exposed as API as it could have some functionality or make it slightly clearer that it's internal with `@ApiStatus.Internal`

Reply:
Ah, I was thinking of replacing the interface mixin with the two accessors that I've done for resource pack tooltips; I think it's much cleaner than the current one